toolchain/check/testdata/array/fail_bound_negative.carbon

```diff
@@ -1,135 +0,0 @@
-// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
-// Exceptions. See /LICENSE for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-// AUTOUPDATE
-// TIP: To test this file alone, run:
-// TIP:   bazel test //toolchain/testing:file_test --test_arg=--file_tests=toolchain/check/testdata/array/fail_bound_negative.carbon
-// TIP: To dump output, run:
-// TIP:   bazel run //toolchain/testing:file_test -- --dump_output --file_tests=toolchain/check/testdata/array/fail_bound_negative.carbon
-
-fn Negate(n: i32) -> i32 = "int.snegate";
-
-// CHECK:STDERR: fail_bound_negative.carbon:[[@LINE+4]]:19: error: array bound of -1 is negative [ArrayBoundNegative]
-// CHECK:STDERR: var a: array(i32, Negate(1));
-// CHECK:STDERR:                   ^~~~~~~~~
-// CHECK:STDERR:
-var a: array(i32, Negate(1));
```

Comment:
Are all the tests still around? Where did this one go?

Reply:
Oh, I searched for the wrong thing. `Negate` is gone because it's written with `-1` now.

Reply:
Yeah, I was assuming this predated relevant support.